modules/apps/fragment/fragment-web/npmscripts.config.js

```diff
@@ -0,0 +1,26 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+module.exports = {
+	build: {
+		bundler: {
+			rules: [
+				{
+					test: '\\.css$',
+					use: ['css-loader'],
```

Comment:
Didn't we make this the default? 🤔

Reply:
Only for `scss`, but there are (I think two) projects that configure it for `.css` alone. I didn't want to move it yet to the base config because it may impact the build times if we begin to process a lot of CSS files that we don't even need.

But we can analyze it too. Should I create a LPS for that?

Reply:
I don't want to add more big changes in this PR. It's already big enough... :grimacing:

Reply:
That's okay, I was just curious... let's get this through!

Reply:
Turns out I was wrong -> https://github.com/liferay/liferay-frontend-projects/blob/master/projects/npm-tools/packages/npm-scripts/src/config/npmscripts.config.js#L56

:thinking: I remember to have checked this locally... :shrug:

Reply:
I'll remove them, then.